packer/scripts/fedora/ganeti.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eux
+
+if [ -e /usr/bin/dnf ] ; then
```

Comment:
@borcean not a terribly good one, but we do intend to execute the file if it exists.